### PR TITLE
Compatibility with phoenix 1.5

### PR DESCRIPTION
**Pull request by afatsini**
Not retro-compatible with previous versions

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -38,12 +38,11 @@ defmodule SimpleAuth.Mixfile do
 
   defp deps do
     [
-      {:phoenix, "~> 1.3.0 or ~> 1.4.0"},
+      {:phoenix, "~> 1.5.0"},
```
Comment (chazsconi) [inline mix.exs]: Can this line just be changed to `~> 1.3.0 or ~1.4.0 or ~1.5.0`?

File: config/config.exs [hunk 2]
```diff
@@ -2,29 +2,4 @@
...
-#
-
-# It is also possible to import configuration files, relative to this
-# directory. For example, you can emulate configuration per environment
-# by uncommenting the line below and defining dev.exs, test.exs and such.
-# Configuration from the imported file will override the ones defined
-# here (which is why it is important to import them last).
-#
-#     import_config "#{Mix.env}.exs"
+config :phoenix, :json_library, Jason
```
Comment (chazsconi) [inline config/config.exs]: Is this needed to run the tests (of which there aren't any 😬 )?

File: mix.exs (hunk 1 above)
Comment (afatsini) [reply mix.exs]: Well its only compabtible with 1.5 now

File: config/config.exs (hunk 2 above)
Comment (afatsini) [reply config/config.exs]: looks like a mandatory config now

**Review by chazsconi — APPROVED** [review]
Thanks a lot @afatsini !

**Pull request merged**